src/_settings.scss

```diff
@@ -156,6 +158,15 @@ $bdrs-small: ceil($bdrs-base / 2) !default;
 $bxsh-spread: 3px !default;
 $bxsh-base: 0 1px $bxsh-spread rgba($c-black, .15) !default;
 
+// Layers (z-index) sorting.
+$z-layers: (
+  'modal-basic': 100
```

Comment:
I would like to have these sorted alphabetically.

Reply:
I think it really makes more sense to have it by z-index value, it gives us a "graphic" status of all the layers sorting, isn't it?

Reply:
:+1:

Reply:
But then, maybe it would be nice to have the tabs prepared to be in the same line for all the values!
```
'modal-basic':                          100 
'cookie-banner':                        50,
'range-datepicker-icon':             2,
```

To have it visually more sorted! :D